lib/index.ts

```diff
@@ -137,7 +137,12 @@ export interface VariableDeclaration extends DeclarationBase {
 
 export interface ExportEqualsDeclaration extends DeclarationBase {
     kind: "export=";
-    target: string;

```

Comment:
Please revert - this is a breaking change (it's legal to inspect an object after it's been created) and somewhat misleading (more complex objects than just 'names' can be `export =`'d)

Reply:
Oh, that is embarrassing, sorry! 😲

<img width="462" alt="screen shot 2018-03-13 at 19 18 22" src="https://user-images.githubusercontent.com/761683/37361570-85f317fc-26f3-11e8-946d-3b1bdcbc37e2.png">
<img width="461" alt="screen shot 2018-03-13 at 19 18 48" src="https://user-images.githubusercontent.com/761683/37361578-884658d4-26f3-11e8-8b74-3158b4099105.png">

Fixed in a9797c9.

Reply:
> more complex objects than just 'names' can be export ='d

I realise that complex objects can be export='d, but this implementation only supports identifier names, right? Therefore I thought it should be named the same as in basically all the other declaration interfaces (i.e. `name`). Nevertheless, as you correctly pointed out, this would be a breaking change, so there's no need to discuss this. I just wanted to understand this statement.